src/common/low_precision_transformations/src/rt_info/quantization_mode_attribute.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include "low_precision/rt_info/quantization_mode_attribute.hpp"
 #include <assert.h>
+#include <sstream>
 
```

Comment:
Why add this header file?

Reply:
removed the header file "#include <ngraph/variant.hpp>" for "quantization_mode_attribute.hpp" caused line 14 "std::stringstream ss;" undefined, so add this header file.

Reply:
> Why add this header file?

removed the header file "#include <ngraph/variant.hpp>" for "quantization_mode_attribute.hpp" caused line 14 "std::stringstream ss;" undefined, so add this header file.